backend/fms_core/services/property_value.py

```diff
@@ -23,4 +23,30 @@ def create_process_properties(properties, processes_by_protocol_id):
         except ValidationError as e:
             errors.append(';'.join(e.messages))
 
+    return (property_values, errors, warnings)
+
+def create_process_measurement_properties(properties, process_measurement):
```

Comment:
I would add a layer of security for us that make sure that each property_type is related to the protocol to which is linked the process_measurement.